lib/src/widgets/property_widgets/color_property.dart

```diff
@@ -63,7 +63,7 @@ class ColorPropertyState extends State<ColorProperty> {
       label: widget.property.name,
       child: ElevatedButton(
         style: ElevatedButton.styleFrom(
-          primary: currentColor,
+          backgroundColor: currentColor,
```

Comment:
Maybe this should be on a different PR? Cause this will need a release

Reply:
It's catch-22, if we don't want to force merge this one, because the analyze on stable will fail without this fix.

Reply:
I can just rename this branch to `fix: Remove usages of deprecated fields and bump to Flutter stable` 😄

Reply:
Renaming the PR works!